lib/commands/doctor/checks/logged-in-user.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const execa = require('execa');
+const errors = require('../../../errors');
+const chalk = require('chalk');
+const os = require('os');
+
+const taskTitle = 'Checking logged in user';
+
+function loggedInUser() {
+    const uid = process.getuid();
+    let ghostuid;
+
+    try {
+        ghostuid = execa.shellSync('id -u ghost').stdout;
+    } catch (e) {
+        // CASE: the ghost user doesn't exist yet, all good in the hood
+        return Promise.resolve();
+    }
+
+    ghostuid = parseInt(ghostuid);
+
+    if (ghostuid === uid) {
+        return Promise.reject(new errors.SystemError({
+            message: 'You can\'t install Ghost with a user called `ghost`. Please use a different user(name).',
+            help: `${chalk.green('https://docs.ghost.org/docs/install#section-create-a-new-user')}`,
+            task: taskTitle
+        }));
+    }
+
+    return Promise.resolve();
+}
+
+module.exports = {
+    title: taskTitle,
+    task: loggedInUser,
+    enabled: (ctx) => !ctx.local && !(ctx.instance && ctx.instance.process.name === 'local'),
+    skip: () => os.platform() !== 'linux',
```

Comment:
should probably get the platform from `ctx.system.platform`, although I'm not 100% either on making it a "skip if not linux". imo it might make more sense to flip the enabled & skip functions?

Reply:
Hmm... I changed it to `ctx.system.platform` (should have known that, as I implemented it 🙈), but I'm not sure about flipping it... In `system-stack` which is also an install task, we have the enabled set the same. See [here](https://github.com/TryGhost/Ghost-CLI/blob/master/lib/commands/doctor/checks/system-stack.js#L66).